### PR TITLE
Refresh avocado man-page (v2)

### PR DESCRIPTION
**Pull request by ana**
Full refresh and update of the avocado man page.

### CODE REVIEW

File: man/avocado.rst [hunk 1]
```diff
@@ -304,79 +306,103 @@ Options for subcommand `diff` (`avocado diff --help`)::
...
+
+    By default, a textual diff report is generated in the standard output.
 
 Options for subcommand `distro` (`avocado distro --help`)::
 
     optional arguments:
       -h, --help            show this help message and exit
       --distro-def-create   Creates a distro definition file based on the path
-                            given
+                            given.
```
Comment (willianrampazzo) [inline man/avocado.rst:321]: This is the only place you added the period. As most places do not have a period, I would follow the pattern and let this line without it.

**Review by willianrampazzo — APPROVED** [review]
So, I see some inconsistency related to the use of periods in the options' descriptions. We need to review them, but this can be done in another PR. In this case, LGTM.

**Comment by ana** [comment]
@willianrampazzo agree, we need to check it directly in the code. This update shows exactly the present output.
What do we prefer periods or not? :-)

**Comment by willianrampazzo** [comment]
> @willianrampazzo agree, we need to check it directly in the code. This update shows exactly the present output.

Okay, that makes sense.

> What do we prefer periods or not? :-)

I don't have a strong preference. It looks like using periods makes it grammatically more correct.

**Review by beraldoleal — APPROVED** [review]
LGTM, thank you @ana .

**Pull request merged**